### ISSUE TITLE
build action

### ISSUE THREAD

**Issue by systemcrash** [ecdye/macSup2Srt#2]
Recommend this action to build binaries (unsigned are acceptable).

https://github.com/mxcl/xcodebuild

**Comment by ecdye** [comment]
Thanks for the suggestion. I will probably implement something like this at some point, but don't have plans to at the moment. If you would like to contribute a change adding this, I would be happy to review and merge a PR.

**Comment by ecdye** [comment]
Target OS is macOS 13 or newer. Minimum XCode version is 15

**Comment by systemcrash** [comment]
No joy. 
```
  xcodebuild: error: Unable to read project 'macSup2Srt.xcodeproj'.
  	Reason: The project ‘macSup2Srt’ cannot be opened because it is in a future Xcode project file format (70). Adjust the project format using a compatible version of Xcode to allow it to be opened by this version of Xcode.
  
  exec: xcodebuild -destination platform=macOS -configuration Release -resultBundlePath build.xcresult build
```

**Comment by ecdye** [comment]
It might be because I was originally using Xcode 16, you could try with that

**Comment by ecdye** [comment]
Well, some things changed and I ended up making this a bit more of a priority so it should be fixed with [8e4628c](https://github.com/ecdye/macSup2Srt/commit/8e4628c56de45dfcc74b1794f9d2c742af77c912). Thanks for bringing this up, sorry to bait and switch.

**Comment by systemcrash** [comment]
OK. Good that you tried. I also experimented a bit. 

So you've managed to automate a build for arm64, but x86_64 is also important :)

I'm not sure about a predictable output directory, but I managed to pass various parameters to `xcodebuild` and get a predictable output by using at least

```sh
... -destination "platform=macOS,arch=$ARCH" install CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO DSTROOT="/Users/runner/work/macSup2Srt/macSup2Srt/Release"
```

Although I did this in a shell script, and not strictly in one of the github actions. The action you chose (I tried it also) doesn't support the destination parameter, I think.
